tests/test_init.py

```diff
@@ -71,47 +71,45 @@
         (qml.init.interferometer_varphi_uniform, base_uni)
         ]
 # Functions returning a list of parameter arrays
-ALL = [(qml.init.cvqnn_layers_all, rpt),
-       (qml.init.interferometer_all, base)]
+INITALL = [(qml.init.cvqnn_layers_all, rpt),
+           (qml.init.interferometer_all, base)]
 
 ###########
 # Shapes
 
-SHP = [(repeat, n_rots), (repeat, n_wires, 3), intf, intf, wrs, wrs,
-       wrs, wrs, wrs, wrs, (n_if,), (n_if,), (n_wires,),
-       (repeat, n_rots), (repeat, n_wires, 3), intf, intf, wrs, wrs,
-       wrs, wrs, wrs, wrs, (n_if,), (n_if,), (n_wires,),
-       ]
-ALL_SHP = [[intf, intf, wrs, wrs, wrs, intf, intf, wrs, wrs, wrs, wrs],
-           [(n_if,), (n_if,), (n_wires,)]]
+SHAPE = [(repeat, n_rots), (repeat, n_wires, 3), intf_block, intf_block, wire_block, wire_block,
```

Comment:
The names are better, but I still find myself struggling to parse easily what these elements refer to.
Suggestion: Can we recast it so the shapes are right next to the corresponding inits? (essentially a hand-written `zip`)
```
INIT_AND_SHAPES = [[(qml.init.random_layers_normal, rnd_rpt_nrml), (repeat, n_rots)], 
                   [(qml.init.strong_ent_layers_normal, rpt_nrml), (repeat, n_wires, 3)],
                   ...
                  ]
```
From that, we could easily make separate `INIT` and `SHAPE` objects programmatically using list comprehensions

Reply:
I will address this in the PR #419, since I changed the signature slightly there. Want to avoid merge conflicts.